modules/pipeline/services/pipelinesvc/secret_test.go

```diff
@@ -71,3 +72,59 @@ func TestAddRegistryLabel(t *testing.T) {
 		})
 	}
 }
+
+func TestReplaceProjectApplication(t *testing.T) {
+	type args struct {
```

Comment:
what if case is "addon-registry.default.svc.cluster.local:5000/", please show me then how it works.

Reply:
addon-registry.default.svc.cluster.local:5000//project-application : tag